src/platform_linux/platform.h

```diff
@@ -65,6 +65,7 @@
 #define FSIZEOF(s, f) sizeof(((typeof(s) *)NULL)->f)
 
 #define MAX_THREADS (64)
+#define INVALID_TID (MAX_THREADS)
```

Comment:
While you are here ... Like we did in a previous round of code-cleanup ... Any chance we can comment the semantic of `MAX_THREADS` ? I guess it's a static hard-limit ... A comment that sheds light on the origins of this limit will be helpful to the new readers.